core/src/display_object/movie_clip.rs

```diff
@@ -2406,6 +2408,72 @@ impl<'gc> MovieClip<'gc> {
 
         unqueued
     }
+
+    /// This unloads the MovieClip.
+    ///
+    /// This means that one frame after this method has been called, avm1_unload and
+    /// transform_to_unloaded_state get called and the MovieClip enters the unloaded
+    /// state in which some attributes have certain values.
+    // TODO: Look at where avm1_unload gets called directly. Does Flash also execute these
+    // calls with one frame delay? Does transform_to_unloaded_state need to get executed
+    // after one frame if the target is a MovieClip? Test the behaviour and adapt the code
+    // if necessary.
+    pub fn unload_movie(&self, context: &mut UpdateContext<'_, 'gc>) {
```

Comment:
Can you rename this to `avm1_unload_movie`?

Reply:
Hey, sorry for the late response, I just wanted to finish the PR first.
But thanks for the suggestion; I can do that! :)

There's also MovieClip::avm1_unload (which is also mentioned in the documentation of this method); do you think there should be further differenciation between these methods if their names are so similar or is it okay like this?